exercises/practice/bob/.docs/instructions.md

```diff
@@ -17,3 +17,11 @@ Bob only ever answers one of five things:
   The convention used for silence is nothing, or various combinations of whitespace characters.
 - **"Whatever."**
   This is what he answers to anything else.
+
+## Examples
+
+input - How are you?
+Bob's response - Sure.
+
+input - I am John
+Bob's response - Whatever.
```

Comment:
The instructions are synced from https://github.com/exercism/problem-specifications/blob/main/exercises/bob/instructions.md. If you think the instructions could be improved, please first discuss this on the forum: https://forum.exercism.org/c/exercism/4

Reply:
ok, sure.

Reply:
Created. https://forum.exercism.org/t/add-example-scenarios-for-the-description-of-the-bob-example-in-ballerina-exercises/5621
Will add a discussions for other scenarios depending on the conclusions of this discussion.

Reply:
Removed the changes made for the instructions